content/applications/marketing/email_marketing/mailing_lists.rst

```diff
@@ -9,128 +9,212 @@ feedback, and more.
+contacts associated with every mailing list.
+
+.. image:: mailing_lists/mailing-list-page.png
    :align: center
-   :alt: View of the new mailing list drop-down on contact form in Odoo Email Marketing.
+   :alt: The Mailing List page in the Odoo Email Marketing application.
+
+The default :guilabel:`Exclude Blacklisted Emails` filter appears in the search bar.
+
+From the :guilabel:`Mailing List Contacts` page, contacts can be created and/or imported, and then
+added to a mailing list.
+
+To add an existing contact to a mailing list, select the desired contact from the list on the
+:guilabel:`Mailing List Contacts` page to reveal their contact form.
 
-The :guilabel:`Create` option quickly creates the mailing list to the contact detail form, and the
-list can be configured at a later date. The :guilabel:`Create and Edit...` option creates the
-mailing list, and reveals a pop-up window, in which the new mailing list can be configured right
-away.
+At the bottom of their contact form, click :guilabel:`Add a line` under the :guilabel:`Mailing List`
+column, locate the desired mailing list from the drop-down menu, and select it.
 
-.. image:: mailing_lists/create-and-edit-mailing-list-popup.png
+.. image:: mailing_lists/contact-form-mailing-list-add.png
    :align: center
-   :alt: View of the create and edit mailing list pop-up in Odoo Email Marketing.
+   :alt: The Add a line for mailing lists on a standard contact form in Odoo Email Marketing.
 
-Link a mailing list to website (Newsletter blocks)
-==================================================
+.. tip::
+   A mailing list can be created directly from a contact form, by typing the name of the new mailing
+   list in the :guilabel:`Mailing List` field. Then, after a new mailing list name has been entered,
+   two options appear on the drop-down menu beneath the new mailing list name.
+
+   From this drop-down menu, select :guilabel:`Create` to create the mailing list and edit it later,
+   or select :guilabel:`Create and edit...` to create and edit the new mailing list right away.
+
+To remove a contact from a mailing list that the contact has been added to, enable the
+:guilabel:`Opt Out` checkbox. If the :guilabel:`Opt Out` checkbox is ticked, the ability to add a
+:guilabel:`Reason` why the contact opted-out is also available.
+
+When/if a contact *has* opted-out of a mailing list, the date they activated their opt-out appears
+in the :guilabel:`Unsubscription Date` column on their contact form.
+
+Lastly, the initial :guilabel:`Subscription Date` can be seen, as well. This field is auto-populated
+with the date and time their subscription to the mailing list has been saved.
+
+Multiple mailing lists can be added to a single contact form.
+
+To delete any mailing list from a contact form, simply click the :guilabel:`🗑️ (trash can)` icon.
+
+Link mailing list to website
+============================

```

Comment:
The proceeding section details how to basically create a newsletter signup _form_ on the website, which is one of a few different ways to connect a mailing list to an Odoo website form.

I would using headings to indicate this as such, and try to include the other use cases here as well (maybe on another PR?) or point to other articles that already cover this information (e.g., add a contact to a mailing list when they reach out through a: sales contact form (product or general inquiries), a helpdesk/support form, etc.)

```suggestion
Link mailing list to website form
=================================

Newsletter signup
-----------------

```